data/local-sqldelight/src/commonMain/sqldelight/com/adammcneilly/pocketleague/sqldelight/LocalEvent.sq

```diff
@@ -20,13 +20,13 @@ ORDER BY startDateUTC;
 selectOngoing:
 SELECT *
 FROM localEvent
-WHERE startDateUTC < DATE() AND endDateUTC > DATE()
+WHERE startDateUTC < :startDateUtc AND endDateUTC > :endDateUtc
 ORDER BY startDateUTC;
 
 selectUpcoming:
 SELECT *
 FROM localEvent
-WHERE startDateUTC > DATE()
+WHERE startDateUTC > :startDateUtc
```

Comment:
It might be worth renaming the queries, since they're no longer compared to now but our provided clock. Could be `selectBetweenDates()` and `selectAfter`, perhaps.